stingray/pulse/pulsar.py

```diff
@@ -395,7 +395,7 @@ def z_n(phase, n=2, norm=1):
                 for k in range(1, n + 1)])
 
 
-def z2_n_detection_level(n=2, epsilon=0.01, ntrial=1):
+def z2_n_detection_level(n=2, epsilon=0.01, ntrial=1, n_summed_spectra=1):
```

Comment:
How was this computed before? I'm mainly curious. How's `n_summed_spectra` different from `n`?

Reply:
`n_summed_spectra` is the number of averaged periodograms, `n` is the "n" in Z^2_n :)

Reply:
I changed the docstrings, hope they are more clear now